### ISSUE TITLE
Changing the duration only when changing the end

### ISSUE THREAD

**Issue by mtc-jed** [Chris1234567899/flutter_time_range_picker#33]
This is a feature proposal.

It would be nice for the widget to be able to keep the duration when updating the start time (ie duration is 8 hours from 8:00 to 16:00, if I slide the start to 10:00 the end slides to 18:00 automatically), but not when updating the end time.

**Comment by Chris1234567899** [comment]
Sounds like a quite specialized feature. Can you describe a good use case for this?

**Comment by mtc-jed** [comment]
Yeah, I know. Main use case is "I came in later/sooner, but I still did the same amount of time". So you just put the right starting time and you're good to go.

**Comment by Chris1234567899** [comment]
I'm not sure if this feature is really that relevant for a wider audience.
I'll leave this issue open for a while to see whether more people are requiring this feature.
In the meanwhile, fell free to make a PR if you want ;-)

**Comment by danielmessi13** [comment]
Up
